react-native-app/views/Confirmation.tsx

```diff
@@ -32,31 +36,37 @@ const Confirmation : React.FC = ({route}) => {
         setError('You must enter confirmation code');
       }
     };
+
+    const resendConfirmationCode = async () => {
+      await Auth.resendSignUp(
+        email
+      )
+    }
   
   return (
     <View style={styles.container}>
-      <Text>Check your email for the confirmation code.</Text>
-      <Input
-        value={authCode}
-        placeholder="123456"
-        onChange={(text : any) => setAuthCode(text)}
-      />
-      <Button onPress={() => confirmSignUp()}>Confirm Sign Up</Button>
-      <Text>{error}</Text>
+      <View style={styles.box}>
+        <Text style={styles.text}>Check your email for the confirmation code.</Text>
```

Comment:
Nit: I've been putting all the text in `assets/constants/text.tsx` to keep everything in one place.